geoips/interfaces/base.py

```diff
@@ -666,14 +667,14 @@ def get_plugin(self, name):
                 f" attempting to access the correct plugin name)"
             ) from resp
         # Convert the module into an object
-        return plugin_module_to_obj(self, name, module)
+        return self.plugin_module_to_obj(name, module)
```

Comment:
Assuming this is what @jsolbrig was looking for,  and the tests still pass,  it looks good to me